app/src/main/java/com/dmonzonis/nookpendium/SortManager.kt

```diff
@@ -0,0 +1,55 @@
+package com.dmonzonis.nookpendium
+
+import android.widget.ImageView
+import android.widget.LinearLayout
+
+class SortManager(private val activity: CollectionActivity) {
+    private var currentSort: Int? = null
+    private var descending: Boolean = false
+    private var imageViewMap = mutableMapOf<Int, ImageView>()
+
+    fun addSortField(sortItem: LinearLayout) {
+        val id = sortItem.id
+        var imgSortOrder = sortItem.findViewById<ImageView>(R.id.imgSortOrder)
+        if (imgSortOrder != null)
+            imageViewMap[id] = imgSortOrder
+
+        // Initially, sort by the first sorting item introduced
+        if (currentSort == null) {
+            currentSort = id
```

Comment:
I don't know about this, I guess the first item in the case of critters would be name? why not ID? fyi the "default" order is the order by internal game id (bitterling is fish id = 0, etc), note that this is the order that they appear in the critterpedia, so if a user wanted to go through their critterpedia and check the checkboxes on the app, the default sorting would be the easiest to follow

Reply:
sure, didn't know they were in a meaningful order, I'll add an ID sort and make it the default